.github/workflows/validate.yml

```diff
@@ -147,6 +147,9 @@ jobs:
     - name: no toolchain in go.mod # See https://github.com/opencontainers/runc/pull/4717, https://github.com/dependabot/dependabot-core/issues/11933.
       run: |
         if grep -q '^toolchain ' go.mod; then echo "Error: go.mod must not have toolchain directive, please fix"; exit 1; fi
+    - name: no exclude nor replace in go.mod
+      run: |
+        if grep -Eq '^(exclude|replace) ' go.mod; then echo "Error: go.mod must not have exclude/replace directive, it breaks go install. Please fix"; exit 1; fi
```

Comment:
Whitespaces may exist before `(exclude|replace)`

Reply:
I pushed a version that matches spaces if they are too.

I think it is okay to handle it, but I'm not sure we will hit it in practice. go-fmt removes the leading spaces in an exclude/replace directive. But `go mod tidy` doesn't seem to remove them. So, I think it is unlikely we will have spaces in practice.

But it's so easy to handle that I pushed a version that does that :)